spec/tic_tac_toe_spec.rb

```diff
@@ -0,0 +1,206 @@
+def find_winner(board)
```

Comment:
Try putting this in a separate file in the `lib/` folder, like `lib/tic_tac_toe.rb`.  You can load the file by using `require` or `require_relative`.